tests/test_config.py

```diff
@@ -214,10 +208,50 @@ def test_entity_customization(self):
 
         self.hass.block_till_done()
 
-        state = self.hass.states.get('test.test')
+        return self.hass.states.get('test.test')
+
+    def test_entity_customization_false(self):
```

Comment:
Ok, I see what you mean. Personally would have done the customization schema (and _old_to_new) in `helpers/customize.py` as well (since its all tied to customize), but keeping this here is ok for now!